http/1.6/modules/ROOT/pages/http-request-ref.adoc

```diff
@@ -29,16 +29,28 @@ In addition to the body of the request, you can configure:
 
 === HTTP Request and Response Size Limits
 
-By default, Mule defines different limits for the HTTP request size, response size, and request/response header size. However, you can change these max values by modifying the corresponding property in the `wrapper.conf` file.
+By default, Mule defines maximum values for the HTTP request size, response size, and request/response header size. However, you can change these values by modifying the following properties in the `wrapper.conf` file:
 
 [%header%autowidth.spread,cols="a,a,a"]
 |===
-| Maximum Allowed Value | Property | Default Value (in bytes)
-| Request/Response Header Size |  `mule.http.headerSectionSize` | `8192`
-| Request Size | `org.glassfish.grizzly.nio.transport.TCPNIOTransport.max-receive-buffer-size` | `1048576` (1MB)
-| Response Size | `org.glassfish.grizzly.nio.transport.TCPNIOTransport.max-send-buffer-size` | `1048576` (1MB)
+| Name | Property | Description | Default Value (in bytes)
+| Request/Response Header Size |  `mule.http.headerSectionSize` | Maximum size in bytes accepted for the HTTP request header section (request line + headers) | `8192`
+| Request Size | `org.glassfish.grizzly.nio.transport.TCPNIOTransport.max-receive-buffer-size` | Maximum size of the receive buffer that controls the number of unread bytes to hold and reflects the maximum number of unacknowledged bytes to send. | `1048576` (1MB)
+| Response Size | `org.glassfish.grizzly.nio.transport.TCPNIOTransport.max-send-buffer-size` | Maximum size of the send buffer that controls the outgoing packages until there is an acknowledgement from the receiving systems. | `1048576` (1MB)
 |===
 
+You can also change the maximum values for the server response headers, server request headers, and client request headers by modifying the following system properties in the `wrapper.conf` file:
+
+[%header%autowidth.spread,cols="a,a,a"]
```

Comment:
You may have to change this table autowidth spread part since the table isn't generating properly I think
Maybe just use [%header%autowidth.spread]?